pybamm/parameters/parameter_values.py

```diff
@@ -163,7 +164,7 @@ def update_from_chemistry(self, chemistry):
                 )
             # Create path to component and load values
             component_path = os.path.join(
-                base_chemistry, component_group + "s", component
+                base_chemistry, component_group.replace(" ", "_") + "s", component
             )
             file_path = self.find_parameter(
                 os.path.join(component_path, "parameters.csv")
```

Comment:
add error message around here if the component group is "anode" or "cathode" with suggested change.

Reply:
Done